DigitalLearningSolutions.Data/Models/Tracker/TrackerEndpointQueryParams.cs

```diff
@@ -6,5 +6,9 @@ public class TrackerEndpointQueryParams
         public int? CustomisationId { get; set; }
         public int? SectionId { get; set; }
         public string? IsPostLearning { get; set; }
+

```

Comment:
Could we match the spacings up here to be the same as above.
The convention is usually no spacing line between properties.

Reply:
yep, done :)